session_0/item.rb

```diff
@@ -41,5 +49,14 @@ def discounted?
   # Note: If there are no items for category, stock price for category
   # should be zero.
   def self.stock_price_by_category(items)
+    stock_prices = Hash.new
+    stock_prices = {1 => 0, 2 => 0, 3 => 0, 4 => 0}
```

Comment:
this is fine but what if i had 100 categories would it still be feasible do this?? 
also what if category codes were not linear what if it was something like
1 => 'Electronics/Gadgets',
21 => 'Books',
13 => 'Furniture',
404 => 'Miscellaneous'